coroutines-extra/src/test/scala/org/coroutines/extra/async-await-tests.scala

```diff
@@ -1,56 +1,29 @@
-package org.coroutines
+package org.coroutines.extra
```

Comment:
We have to keep the async-await test in the base project, as they are primarily a robustness test for our coroutine implementation. If there are a additional tests that specifically test functionality in async-await, then you should create a new file in the `extra` project and put those additional tests there.